benchmarks/src/main/java/org/elasticsearch/benchmark/compute/operator/EvalBenchmark.java

```diff
@@ -61,6 +61,11 @@
 public class EvalBenchmark {
     private static final int BLOCK_LENGTH = 8 * 1024;
 
+    static final DriverContext driverContext = new DriverContext(
+        BigArrays.NON_RECYCLING_INSTANCE,
+        BlockFactory.getInstance(new NoopCircuitBreaker("noop"), BigArrays.NON_RECYCLING_INSTANCE)
+    );
+
     static {
```

Comment:
This needs to be loaded first before it's used by calling `run` in the `static` block below.